lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -405,7 +400,12 @@ def launch_all_nodes
           instance = create_instance(host, ami_spec, nil)
           instances.push({:instance => instance, :host => host})
         end
+        # Add metadata tags to each instance
```

Comment:
I was planning to rebase [beaker-aws](https://github.com/puppetlabs/beaker-aws) on the master branch of beaker once we merge #1434. So I would say if you are planning to work on this targeting beaker-aws, wait till the end of the day to have up to date files on beaker-aws.